courts_db/data/courts.json

```diff
@@ -17440,7 +17426,7 @@
             }
         ],
         "name": "Decisions of the Federal Communications Commission",
-        "level": "",
+        "level": null,
```

Comment:
This is philosophical, but generally I follow Django's approach to empty strings, which is:

> In most cases, it’s redundant to have two possible values for “no data;” the Django convention is to use the empty string, not NULL. 

(https://docs.djangoproject.com/en/3.0/ref/models/fields/#django.db.models.Field.null)

I've argued about it before, but I think the convention makes enough sense that it's worth it just to be consistent throughout FLP. I guess I'd listen to opposing views, but I usually appreciate knowing that strings with no value are `""` instead of either `null` or `""`.

Reply:
Something tells me courts-db is already using `null` in this way though....hrm. If that's the case, I feel ambivalent about changing all of courts-db to use `""` instead of `null`, though I still kind of feel like it'd be worth it.

Reply:
Oof...I'm not well acquainted with Django-land, but that seems like a really weird convention. To me, empty string means empty string, not absence of data. How does that even really help? You're just doing a different check (`if x == ""` instead of `if x is None`).

In any event, for the `level` element here, it is constrained by an `enum` so you would never have to guess whether you're getting an empty string or a `null`. The `enum` in the schema tells you only one of those is valid. See: https://github.com/freelawproject/courts-db/pull/10/files#diff-11aaec965ced488b7af5aa03d35c580dR32-R43

Courts-DB was in fact using a lot of empty strings, but I changed them to `null`s in a7297bc78c13e513ca2fb1669b620f211d12351f.

Reply:
> In any event, for the level element here, it is constrained by an enum so you would never have to guess whether you're getting an empty string or a null

That assumes you remember that there's a schema and what's in it. The nice thing about sticking with the convention is that you always know that "no data" is `""`, no matter where you are in the code base. 

> You're just doing a different check (if x == "" instead of if x is None)

In Django, you can almost always do `x == ""` precisely because the convention is there. Without it, you have to do `x == "" or x is None`, because "no data" can have multiple values.

If I win you over a bit, let's switch it back. If you feel like I'm wrong, and see no value in Django's approach, very well.